forms-flow-api/src/formsflow_api/services/factory/keycloak_client_service.py

```diff
@@ -22,6 +22,7 @@ def __init__(self):
         """Initialize client."""
         self.client = KeycloakAdminAPIService()
         self.user_service = UserService()
+        self.group_service = KeycloakGroupService()
```

Comment:
Instead of adding KeycloakGroupService as a member, it's better to set the KeycloakGroupService as base class for the KeyCloakClient service, so that Client Service will inherit Group Service functions, and override only for the ones which are needed.

Reply:
Done